src/js/footer/index.js

```diff
@@ -1,3 +1,4 @@
 // All the footer js files should be imported and exported from here
 import './helper';
+import './updateUrlAsPerLanguage'
 import './footer/index.js';
```

Comment:
@mayuran-deriv there is no such file exists. 
please check this.
```suggestion
```

Reply:
hey ako src/js/footer/updateUrlAsPerLanguage/index.js 
there is a file here

Reply:
this is already in footer/index.js file

Reply:
hey @ali-hosseini-deriv there is a file 
https://github.com/deriv-com/webflow-deriv-com-scripts/blob/master/src/js/footer/footer/index.js

its eduard's code

Reply:
I see, that's the thing that I didn't notice on the other pr. it was like inception to me :-D 
let me merge this 
will ask @eduard-deriv to rename that directory